invenio_app_ils/ui/.env

```diff
@@ -0,0 +1,6 @@
+REACT_APP_JWT_TOKEN='token'
```

Comment:
You can remove `REACT_APP_JWT_TOKEN` and `REACT_APP_JWT_TOKEN_EXPIRATION`, I think they are not used at all... WDYT?

Reply:
I left it for the same reason as below :)